### DISCUSSION TITLE
GROUP does not work in AppWidget

### DISCUSSION THREAD

**Discussion by SashaCX75** [zepp-health/zeppos-forum#267] [Developer Feedback]
If you create a group on the AppWidget page and place an item in it, nothing happens. The logs do not show any errors, but the widget is not created.

**Comment by MyoungXUE** [comment]
Hello @SashaCX75, the `GROUP` widget cannot be used in the shortcut card. For details, please refer to this part of the document.

https://docs.zepp.com/docs/guides/framework/device/secondary-widget/#zepp-os-20-open-related-js-api

![image](https://github.com/zepp-health/zeppos-forum/assets/107526244/04a502e3-f072-44de-a321-646ad6b2afd2)

**Reply by SashaCX75** [reply]
Thanks, I got it. 
I was just looking for a description of the restrictions on the page and the description of the GROUP widget. And it says that widgets can not be nested, and there is nothing about this restriction.

**Reply by MyoungXUE** [reply]
It is also very necessary to indicate this in the restrictions of Group components, and I will add it as soon as possible.
